logos/src/logos/capacity/capacity_planner.py

```
@@ -1951,7 +2109,37 @@ def _compute_demand_actions(
+                    " for another worker this cycle",
+                    self._facade.get_provider_name(provider_id) or provider_id,
+                    model_name,
+                )
+                continue
             model_lanes = lanes_by_model.get(model_name, [])
+            # Phase 3.4: per-lane VRAM ledger gate — under v2, only skip this
+            # model if an in-flight reservation overlaps the GPUs its target
+            # lane(s) would use. Models targeting disjoint GPUs proceed.
+            if self._eviction_gate_v2 and committed > 0:
+                target_gpus: frozenset[int] = frozenset()
+                for l in model_lanes:
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)